platform/pensando/sonic-platform-modules-dpu/dpu/service/midplane-network-dpu.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Midplane network service
```

Comment:
didn't we have a sonic service to achieve this? we do we need such service per vendor?

Reply:
@lguohan, we use int_mnic0 interface for dpu pensando firmware docker image, which communicates over pcie connection. Renaming it will cause issues in functionalities of our firmware dpu container along with syncd and swss. Hence I am using our interface name instead of generic service which uses eth0-midplane, and dpu container should be up before it starts.

Reply:
@SahilChaudhari can we use alias for int_mnic0 ?

Reply:
The mgmt pcie interface to the nic is named as int_mnic and we cannot rename it as it used by different processes running inside polaris docker. We tried aliasing it to eth0-mgmt but that did not work.